dockerfile/api-core/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM docker.wso2.com/base-c5
-COPY wso2apim-3.0.0-m5.zip .
-RUN unzip wso2apim-3.0.0-m5.zip -d /opt
-COPY deployment.yaml /opt/wso2apim-3.0.0-m5/conf/
+MAINTAINER dev@wso2.org
+
+WORKDIR /opt
+COPY wso2apim-3.0.0-m6.zip /opt
+COPY deployment.yaml /opt
+
+RUN unzip wso2apim-3.0.0-m6.zip && \
+    chown -R wso2:wso2 /opt/wso2apim-3.0.0-m6 && \
+    rm -rf /opt/wso2apim-3.0.0-m6.zip && \
+    cp /opt/deployment.yaml /opt/wso2apim-3.0.0-m6/conf
+
 EXPOSE 9292
-RUN chgrp -R 0 /opt && chmod -R g+rwX /opt
-CMD ["/opt/wso2apim-3.0.0-m5/bin/carbon.sh"]
+
+CMD ["/opt/wso2apim-3.0.0-m6/bin/carbon.sh"]
```

Comment:
Maybe we can introduce ARGs for dockerfiles. Otherwise users have to change multiple places when doing a change. https://github.com/wso2/kubernetes-apim/blob/2.1.0/base/apim/Dockerfile#L6